modules/apps/static/portal-file-install/portal-file-install-impl/src/main/java/com/liferay/portal/file/install/internal/properties/TypedProperties.java

```diff
@@ -209,19 +208,6 @@ public boolean nextProperty() throws IOException {
 
 			value = InterpolationUtil.substVars(value.trim());
 
-			if (value.startsWith("LIFERAY_")) {
-				value = InterpolationUtil.substVars(
-					GetterUtil.getString(System.getenv(value)));
-
-				matcher = _propertyValuePattern.matcher(value);
```

Comment:
@mtambara I think this is the only usage of the field, you'll need to remove it.